src/components/molecules/AssetTeaser.tsx

```diff
@@ -37,7 +39,17 @@ const AssetTeaser: React.FC<AssetTeaserProps> = ({ ddo }: AssetTeaserProps) => {
           </Dotdotdot>
           <Publisher account={owner} minimal className={styles.publisher} />
         </header>
-        {isCompute && <div className={styles.accessLabel}>Compute</div>}
+
+        <div className={styles.typeDetails}>
+          <Tooltip content={accessType === 'access' ? 'Download' : 'Compute'}>
+            {accessType === 'access' ? (
+              <Download className={styles.icon} />
```

Comment:
since these icons have no accompanying text but still convey information we should add some ARIA attributes for accessibility reasons, in this case: `role="img" aria-label="Download"`